src/main/resources/META-INF/plugin.xml

```diff
@@ -36,6 +36,12 @@
                 <override-text place="GitMacheteContextMenu"/>
             </action>
 
+            <action id="GitMachete.NextBranchCheckoutAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.contextmenu.NextBranchCheckoutAction">
+                <override-text place="GitMacheteContextMenu"/>
```

Comment:
TBD whether these actions should even be in the context menu... actually it'd only make sense if they have predefined keyboard shortcuts, and then the menu will serve as kinda-tip for the users that such checkout actions and their keyboard shortcuts are available at all

Reply:
CC @mkondratek

Reply:
Note that as for now we've got just one predefined keyboard shortcut: `Ctrl+Alt+Shift+M` for opening the Git Machete tab

Reply:
Thank you Pawel, so if this is the desired functionality, I would implement `previous` and `up` too.

Reply:
Yes, the general direction is okay, details TBD

Reply:
Pls think over the possible combinations of keyboard shortcuts 🤔 for Mac/non-Mac

Reply:
Finding key combinations on Mac is very difficult. Whatever key combination I use, it says, it is already taken. except shift+Letter sometimes.

Reply:
Note that many shortcuts are taken by somewhat niche actions (limited to the scope of a specific plugin etc.)... ofc we shouldn't try taking over any well-known, global shortcuts (like, `Ctrl+Shift+Alt+N` for _Next branch_ would be a very bad idea 😅)

Reply:
Ctrl+shift+option+M does nothing for me :(

Reply:
On Mac it's ⌘⌥⇧M: https://github.com/VirtusLab/git-machete-intellij-plugin/#where-to-find-the-plugin-tab

Reply:
we can try vim direction keys (jk) 😆 

![image](https://user-images.githubusercontent.com/19799111/192350760-87d6f84a-a35b-40f1-aae7-5942a7fe622c.png)

Reply:
> we can try vim direction keys (jk) laughing

Huh nope 😅 I mean, even ASDF would be better ;D knowledge of vim among the developers (not counting the devops/sysadmin crowd) is rather low... for most users, this choice will look rather random/arbitrary ☹️

Reply:
TBH I myself despite using `vim` quite extensively back in early 2010s, I've never got used to HJKL... always the arrows